trading_calendars/common_holidays.py

```diff
@@ -1,12 +1,13 @@
-from pandas.tseries.holiday import Holiday, Easter
+from pandas.tseries.holiday import Holiday, GoodFriday, Easter, EasterMonday  # noqa
```

Comment:
to be consistent with the rest of this file, should we make factory functions for Good Friday and Easter Monday? Otherwise we can just import them from pandas.tseries.holiday where needed

Reply:
I'll make factory functions for them. I'd like to have them imported in this file.

Reply:
jk that seems extraneous. I'll just move them